src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -2438,6 +2436,27 @@ DistributionColumnUsesGeneratedStoredColumn(TupleDesc relationDesc,
 }
 
 
+/*
+ * DistributionColumnUsesGeneratedIdentityColumn returns whether a given relation uses
+ * GENERATED (ALWAYS|BY DEFAULT) AS (...) IDENTITY on distribution column
+ */
+static bool
+DistributionColumnUsesGeneratedIdentityColumn(TupleDesc relationDesc,
```

Comment:
Given that it's only used [here](https://github.com/citusdata/citus/pull/6591/files#r1066842072), you can delete this function definition altogether.